adapters/rack/lib/opentelemetry/adapters/rack/adapter.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Rack
+      # The Adapter class contains logic to detect and install the Rack
+      # instrumentation adapter
+      class Adapter < OpenTelemetry::Instrumentation::Adapter
+        install do |config|
+          require_dependencies
```

Comment:
Mental note, this is the 3rd time I've seen this. I'm starting to think this should be an optional part of the DSL.